2-6/Dockerfile

```diff
@@ -1,33 +1,52 @@
 FROM ruby:2
 
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
+
+# gpg keys listed at https://github.com/nodejs/node#release-team
+RUN set -ex \
+  && for key in \
+    9554F04D7259F04124DE6B476D5A82AC7E37093B \
+    94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
+    FD3A5288F042B6850C66B31F09FE44734EB7990E \
+    71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+    DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+    C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+    56730D5401028683275BD23C23EFEFE93C4CFFFE \
+  ; do \
+    gpg --keyserver pgp.mit.edu --recv-keys "$key" || \
+    gpg --keyserver keyserver.pgp.com --recv-keys "$key" || \
+    gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" ; \
+  done
+
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_MAJOR_VERSION 6
```

Comment:
Dam. Looks like i remove it. I just tried to export Node version from RUN command to default node environmental variable NODE_VERSION. But it doesnt work, because every command is separate bash. 

I will return that ENV. But i think name should be changed to NODE_MAJOR based on this Docker best practices and example: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#env Ruby Docker also using RUBY_MAJOR without _VERSION suffix.

What do you think?